README.md

```
@@ -30,6 +30,30 @@ To set up the project on your local machine, follow these steps:
   npm run dev
   ```
 
+## Desktop Application Development (Tauri)
+
+CircuitVerse Frontend Vue supports building a cross-platform desktop application using **Tauri**.
+
+### Prerequisites
+
+- Node.js (LTS recommended)
+- npm
+- Rust (required by Tauri): https://www.rust-lang.org/tools/install
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)